tests/conftest.py

```diff
@@ -5,6 +5,19 @@
 import subprocess
 import tempfile
 
+from pytest_server_fixtures.http import SimpleHTTPTestServer
+
+@pytest.yield_fixture(scope='function')
+def simple_http_server():
+    def hooray_its_up(self):
+        return True
+    original_check_server_up = SimpleHTTPTestServer.check_server_up
```

Comment:
This works, but it would be cleaner to use [pytest-mock](https://github.com/pytest-dev/pytest-mock/), instead:
```python
def simple_http_server(mocker):
    mocker.patch.object(SimpleHTTPTestServer, "check_server_up", return_value=True)
    with SimpleHTTPTestServer() as s:
        s.start()
        return s
```

You don't even need to make it a `yield_fixture`, since pytest-mock will automatically undo the mock at the end of the test!